client/router.js

```diff
@@ -369,3 +427,56 @@ Router.prototype._updateCallbacks = function () {
 
 Router.prototype._page = page;
 Router.prototype._qs = qs;
+
+function shouldCallTrigger(current, fn) {
```

Comment:
We've other ways to test.
If you feel router getting bigger, try to move them to a util namespace.
I really don't like to have private methods hidden inside like this.
(I sometimes do it although if I were lazy)